source/operations/external-iam/configure-ad-ldap-external-identity-management.rst

```diff
@@ -99,40 +101,83 @@ An AD/LDAP user with no assigned policy *and* with membership in groups with no
 
    1) Set the Active Directory / LDAP Configuration Settings
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-   You can configure the AD/LDAP provider using either
-   environment variables *or* server runtime configuration settings. Both
-   methods require starting/restarting the MinIO deployment to apply changes. The
-   following tabs provide a quick reference of all required and optional
-   environment variables and configuration settings respectively:
+   Configure the AD/LDAP provider using one of the following:
+
+   * MinIO Client
+   * environment variables
+   * MinIO Console
+
+   All methods require starting/restarting the MinIO deployment to apply changes.
+
+   The following tabs provide a quick reference for the available configuration methods:
 
    .. tab-set::
 
+      .. tab-item:: MinIO Client
+
+         MinIO supports specifying the AD/LDAP provider settings using :mc:`mc idp ldap` commands.
+
+         For distributed deployments, the :mc:`mc idp ldap` command applies the configuration to all nodes in the deployment. 
+
+         The following example code sets *all* configuration settings related to configuring an AD/LDAP provider for external identity management.
+	 The minimum *required* settings are:
+
+         - :mc-conf:`server_addr <identity_ldap.server_addr>`
+         - :mc-conf:`lookup_bind_dn <identity_ldap.lookup_bind_dn>`
+         - :mc-conf:`lookup_bind_password <identity_ldap.lookup_bind_password>`
+         - :mc-conf:`user_dn_search_base_dn <identity_ldap.user_dn_search_base_dn>`
+         - :mc-conf:`user_dn_search_filter <identity_ldap.user_dn_search_filter>`
+
+        .. code-block:: shell
+           :class: copyable
+
+	   mc idp ldap add ALIAS                                                   \
+	      server_addr="ldaps.example.net:636"                                  \
+              lookup_bind_dn="CN=xxxxx,OU=xxxxx,OU=xxxxx,DC=example,DC=net"        \
+	      lookup_bind_password="xxxxxxxx"                                      \
+	      user_dn_search_base_dn="DC=example,DC=net"                           \
+	      user_dn_search_filter="(&(objectCategory=user)(sAMAccountName=%s))"  \
+	      group_search_filter= "(&(objectClass=group)(member=%d))"             \
+	      group_search_base_dn="ou=MinIO Users,dc=example,dc=net"              \
+              enabled="true"                                                       \
+              sts_expiry="1h"                                                      \
+              username_format="uid=%s,cn=miniousers,dc=myldapserver,dc=net,userPrincipalName=%s,cn=miniousers,dc=myldapserver,dc=net"                                            \
+              tls_skip_verify="off"                                                \
+              server_insecure=off                                                  \
+              server_starttls="off"                                                \
+	      comment="Test LDAP server"
```

Comment:
This example contains all the relevant envvars/configuration key settings. Is this/these sample values reasonable?

Reply:
Also: better way to format the long line for `username_format`?

Reply:
I think those are fine for sample values - we cannot reasonably dig int AD/LDAP syntax here.

As far as formatting...not really :grimacing: 

They'll have to scroll, and that is the life that AD/LDAP admins are committed to.

Reply:
I think the long line is fine as is. There will be a horizontal scroll bar in the rendered site for people who want to see the whole thing.